docs/CONFIGURATION.md

```diff
@@ -0,0 +1,270 @@
+## For English press [here](#configuration)
+# Konfiguracja
+
+Ten dokument zawiera przegląd wartości konfigurowalnych w pliku [`values.yaml`](../charts/kam/values.yaml) dla chartu Helm. Te wartości mogą być dostosowywane w celu skonfigurowania wdrożenia aplikacji. W celu uzyskania szczegółowych informacji jak wdrożyć aplikację, odwołaj się do pliku [DEPLOYMENT.md](./DEPLOYMENT.md).
+
+## Ogólna konfiguracja
+
+- **nameOverride**: Nadpisuje nazwę chartu. Przydatne, jeśli chcesz wdrożyć wiele instancji chartu z różnymi nazwami.
+- **fullnameOverride**: Całkowicie nadpisuje wygenerowaną nazwę chartu.
+- **podAnnotations**: Adnotacje do dodania do zasobów pod.
+- **podLabels**: Etykiety do dodania do zasobów pod.
```

Comment:
Maybe I'm not the best person to point this out, but it sounds a little bit wierd do ** do

Reply:
It is perfectly legal move in polish